quickstarts/elastic-knn-search/config.yml

```diff
@@ -0,0 +1,55 @@
+
+slug: elastic-knn-search
+description: |
+  ## Why should you monitor your usage of Elastic KNN Search?
+
+  Monitor your vector searches on Elastic KNN Search to get visibility on what you send to Elastic KNN Search, responses retrieved from Elastic KNN Search, latency, usage and errors. 
+
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+ 
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
+
+
+  ### What’s included in this quickstart?
+
+  New Relic LangChain monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your LangChain app. These reports include:
+  
+  - Vector searches
+  - Alerts for errors, search per vector store, and response time
+  - Identify popular queries, sources, and content
+
+summary: |
+  Monitor your Vector search's performance and quality with New Relic Elastic KNN Search quickstart.
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title: Elastic KNN Search
+documentation:
+  - name: LangChain Vector Database integration documentation
+    description: |
+      Implement monitoring and instrumentation for your Vector store, and ensure that your observability data is integrated into New Relic for effective performance analysis and insights.
+    url: https://github.com/newrelic/nr-openai-observability
+dataSourceIds:
+  - langchain-vectordb
+keywords:
+  - elastic knn search
+  - k-nearest neighbors
+  - KNN
+  - vector search
+  - nearest neighbor search
+  - machine learning
+  - artificial intelligence
+  - AI
+  - ML
+  - data science
+  - big data
+  - elasticsearch
```

Comment:
```suggestion
  - elasticsearch
  - elastic search
```

Reply:
Hi @rahulbasu-nr,
I have made changes based on your comments. Kindly review it.

Thanks...